sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/SensorContext.java

```diff
@@ -62,6 +63,11 @@
    */
   InputModule module();
 
+  /**
+   * @since 5.5
+   */
+  SonarQubeVersion getSonarQubeVersion();
```

Comment:
Do you think that this shortcut method deserves to be documented in the example of `SonarQubeVersion` ?